src/crimson/os/seastore/backref/backref_tree_node.h

```diff
@@ -26,11 +26,11 @@ struct backref_map_val_t {
     laddr_t laddr,
     extent_types_t type)
     : len(len), laddr(laddr), type(type) {}
+
+  bool operator==(const backref_map_val_t& rhs) const noexcept {
+    return len == rhs.len && laddr == rhs.laddr;
```

Comment:
Seems we need to consider type in the comparison. @xxhdx1985126 is it correct?

Anyway, LGTM in this PR.

Reply:
yeah, let's leave it to another PR.